pep-0723.rst

```diff
@@ -156,14 +157,19 @@ version of ``pyproject.toml``, which is called ``Cargo.toml``:
         println!("Did our date match? {}", re.is_match("2014-01-01"));
     }
 
-One important thing to note is that the metadata is embedded in a comment
-mostly for introspection since Rust documentation is generated from comments.
-Another is that users rarely edit dependencies manually, but rather use their
-Cargo package manager.
+One important thing to note is that the metadata is embedded in a doc-comment.
+`Other syntaxes`__ are under consideration, including using attributes which
+are somewhat like a syntatically recognized equivelant of dunder variables,
+with the key difference between Rust's choice and this PEP being that any valid
+Rust syntax will be allowed, requiring one of the Rust syntax parsers to work
+with it, like `syn`__
 
-We argue that our choice, in comparison to the Rust format, is easier to read
-and provides easier edits for humans by virtue of the contents starting at the
-beginning of lines so would precisely match the contents of a
+__ https://github.com/epage/cargo-script-mvs/blob/main/0000-cargo-script.md#embedded-manifest-format
+__ https://crates.io/crates/syn
+
+We argue that our choice, in comparison to the doc-comment approach, is easier
```

Comment:
As above, if we could provide a reference link here for "doc-comment" if possible.
```suggestion
We argue that our choice, in comparison to the doc-comment approach, is easier
```